analytics_data_api/v0/serializers.py

```diff
@@ -449,6 +449,12 @@ def get_videos_viewed(self, obj):
         return obj.get('videos_viewed', 0)
 
 
+class EnterpriseLearnerEngagementSerializer(serializers.ModelSerializer):
+    class Meta(object):
+        model = models.ModuleEngagement
+        fields = '__all__'
```

Comment:
Do we need all fields here? The ticket mentions the following:
course_id
username
date
event
count
entity_type

Reply:
api consumer can pick the interested fields and ignore the rest.